config/dropped_strains.txt

```diff
@@ -26,44 +26,44 @@ DENV2/TRINIDAD_AND_TOBAGO/NA/1953
 DENV4/MALAYSIA/P215/1975
 DENV4/MALAYSIA/P514/1975
 DENV4/MALAYSIA/P731120/1973
-D2Sab2015 # miscategorized
-QML22 # miscategorized
-DAK_Ar_A1247 # sylvatic
```

Comment:
Git history can tell us a little (only what's included in the commit messages). Those lines come from:

1. 3e0f9feaa4d1799cdda6cf839e03d09390f39c53
2. caa787249feadf1a7893662f6ea30230aeb07c46
3. 6e6bd7d00664fecf85a4b1f46073a8cdff02e0e1

It looks like those may predate the use of data from ViPR, given 081f9f82d971e75848ac1967bf6d841e05428545 is the first commit which adds dropped strains not in that earlier format.

I would say drop them, since there's zero chance they can apply now given the switch from strain name → accession. (Right?)